UI/window-projector.cpp

```diff
@@ -659,22 +660,22 @@ static int getSourceByPosition(int x, int y)
 	int     minY  = 0;
 	int     maxX  = cx;
 	int     maxY  = cy;
-	int     halfX = cx / 2;
-	int     halfY = cy / 2;
+	int     pvwpgmX = cx / 2;
+	int     pvwpgmY = cy / 2;
```

Comment:
I am unsure why these names were changed for these variables.  The name you named them to makes them seemingly less readable, and add unnecessary changes to this commit/diff that sort of confuse me here, and have no mention in the commit message.